Jenkinsfile

```diff
@@ -0,0 +1,71 @@
+node('linux && docker') {
+  checkout scm
+  def tag = env.TAG_NAME
+
+  withEnv([
+    'npm_config_cache=npm-cache'
+  ]){
+    withDockerContainer(image: 'nikolaik/python-nodejs:python3.8-nodejs10', args: '-u=root') {
+      stage('Install') {
+        sh 'yarn install'
+      }
+
+      stage('Build') {
+        sh '. ./read.version.sh'
+        sh 'echo $PACKAGE_JSON_VERSION'
+        sh 'yarn run injectTag'
+        sh 'yarn run build'
+
+        if (tag) {
+          sh 'yarn run minimize'
+        }
+      }
+
+      stage('Install Chrome') {
+        sh "wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -"
+        sh "echo 'deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main' | tee /etc/apt/sources.list.d/google-chrome.list"
+        sh "apt-get update"
+        sh "apt-get install -y google-chrome-stable fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-kacst fonts-freefont-ttf libxss1 libxtst6 --no-install-recommends"
+        sh "google-chrome --version"
+      }
+
+      stage('Test') {
+        sh 'yarn run unitTests'
+        
+        if (tag) {
+          sh 'yarn run accessibilityTests'
+        }
+
+        sh 'set +e'
+        // sh 'yarn run uploadCoverage'
+        sh 'set -e'
+        sh 'yarn run validateTypeDefinitions'
+      }
+
+      stage('Docs') {
+        sh 'if [[ "x$TAG_NAME" != "x" && $IS_PULL_REQUEST_PUSH_BUILD = false ]]; then bash ./deploy.doc.sh ; fi'
```

Comment:
`$IS_PULL_REQUEST_PUSH_BUILD` appears to be undefined here. It's a variable I originally added in the travis file.

I think `$TAG_NAME` may also not exist, since it's generated by travis.

Reply:
Indeed, you are right. I will comment the line out to test the full-loop and deploy a beta package to start testing. In the meantime, I will work on this in a separate jira.